node.js/week3/homework/warmup/index.js

```diff
@@ -1,6 +1,123 @@
 const express = require("express");
 const app = express();
 
+
+app.use(
+    express.urlencoded({
+        extended: true
+    })
+)
+app.use(express.json());
+
+
 app.get("/", (req, res) => res.send("nodejs week3 homework"));
 
+app.get("/calculator/add", (request, response) => {
+    console.log(request.query);
+    try {
+        const paramValueArray = Object.values(request.query)
+            .flatMap(x => x)
+            .map(x => parseInt(x));
```

Comment:
Good idea to use flatMap. I actually think you could simplify this into 1 step:

```suggestion
            .flatMap(x => parseInt(x))
```

(I haven't tested it, so I could be wrong)

Reply:
Thanks for the suggestion. That's perfect, and as It combines the steps of first mapping over the array with map() & then calling flat(), it will work.